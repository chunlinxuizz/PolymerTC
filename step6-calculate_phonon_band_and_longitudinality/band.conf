DIM = 1 2 4
BAND = 0 0 0 -0.50238636  0.83653954  0.66992366
BAND_POINTS = 101
EIGENVECTORS = .TRUE.