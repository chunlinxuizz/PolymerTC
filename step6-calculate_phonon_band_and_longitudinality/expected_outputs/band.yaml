calculator: cp2k
length_unit: angstrom
nqpoint: 101    
npath: 1      
segment_nqpoint:
- 101
reciprocal_lattice:
- [  -0.00000000,  -0.00000000,  -0.05336908 ] # a*
- [  -0.00000000,  -0.11246689,  -0.01141439 ] # b*
- [  -0.18050379,  -0.01402370,  -0.02576898 ] # c*
natom: 108    
lattice:
- [     2.527238672900000,     1.901683826800000,   -18.737440315099999 ] # a
- [     0.690798905000000,    -8.891505596000000,     0.000000000000000 ] # b
- [    -5.540050000000000,     0.000000000000000,     0.000000000000000 ] # c
points:
- symbol: C  # 1
  coordinates: [  0.514139999999706,  0.392940000002042,  0.912880000002919 ]
  mass: 12.010700
- symbol: C  # 2
  coordinates: [  0.443940000000774,  0.329780000002495,  0.855080000007546 ]
  mass: 12.010700
- symbol: C  # 3
  coordinates: [  0.431270000000364,  0.219960000009961,  0.652080000002380 ]
  mass: 12.010700
- symbol: C  # 4
  coordinates: [  0.493400000001583,  0.198700000002354,  0.548480000006857 ]
  mass: 12.010700
- symbol: C  # 5
  coordinates: [  0.505540000000232,  0.101180000010981,  0.342120000002516 ]
  mass: 12.010700
- symbol: C  # 6
  coordinates: [  0.570449999997396,  0.088760000010364,  0.255080000011585 ]
  mass: 12.010700
- symbol: C  # 7
  coordinates: [  0.494640000002078,  0.913720000001708,  0.972600000008355 ]
  mass: 12.010700
- symbol: C  # 8
  coordinates: [  0.564150000001939,  0.983340000002504,  0.048680000011889 ]
  mass: 12.010700
- symbol: C  # 9
  coordinates: [  0.488340000001284,  0.808280000005294,  0.766200000011604 ]
  mass: 12.010700
- symbol: C  # 10
  coordinates: [  0.553259999998280,  0.795859999997699,  0.679159999999281 ]
  mass: 12.010700
- symbol: C  # 11
  coordinates: [  0.565420000001930,  0.698340000004756,  0.472840000010285 ]
  mass: 12.010700
- symbol: C  # 12
  coordinates: [  0.627559999997643,  0.677060000001616,  0.369359999996315 ]
  mass: 12.010700
- symbol: C  # 13
  coordinates: [  0.614920000002066,  0.567220000001874,  0.166360000007271 ]
  mass: 12.010700
- symbol: C  # 14
  coordinates: [  0.544719999997797,  0.504079999999846,  0.108480000005201 ]
  mass: 12.010700
- symbol: C  # 15
  coordinates: [  0.700960000001468,  0.752800000007616,  0.465120000008837 ]
  mass: 12.010700
- symbol: C  # 16
  coordinates: [  0.739100000000512,  0.666160000006466,  0.666440000000172 ]
  mass: 12.010700
- symbol: C  # 17
  coordinates: [  0.803420000002261,  0.756779999998996,  0.824600000002209 ]
  mass: 12.010700
- symbol: C  # 18
  coordinates: [  0.840340000000473,  0.662840000006902,  0.015079999996561 ]
  mass: 12.010700
- symbol: C  # 19
  coordinates: [  0.898449999999915,  0.754100000005027,  0.200680000005553 ]
  mass: 12.010700
- symbol: C  # 20
  coordinates: [  0.934510000001916,  0.657800000004241,  0.389040000005154 ]
  mass: 12.010700
- symbol: C  # 21
  coordinates: [  0.992360000000393,  0.748439999999298,  0.576639999999594 ]
  mass: 12.010700
- symbol: C  # 22
  coordinates: [  0.028830000000836,  0.652260000009141,  0.764280000004230 ]
  mass: 12.010700
- symbol: C  # 23
  coordinates: [  0.087019999998933,  0.743360000007147,  0.950200000007003 ]
  mass: 12.010700
- symbol: C  # 24
  coordinates: [  0.123949999996977,  0.647620000005015,  0.137400000010447 ]
  mass: 12.010700
- symbol: C  # 25
  coordinates: [  0.182369999996542,  0.738660000005126,  0.322720000007017 ]
  mass: 12.010700
- symbol: C  # 26
  coordinates: [  0.219029999999127,  0.642180000004843,  0.509320000007099 ]
  mass: 12.010700
- symbol: C  # 27
  coordinates: [  0.276889999997436,  0.732060000010124,  0.698280000006154 ]
  mass: 12.010700
- symbol: C  # 28
  coordinates: [  0.312739999997632,  0.632120000005107,  0.881079999997994 ]
  mass: 12.010700
- symbol: C  # 29
  coordinates: [  0.357870000001877,  0.144220000005102,  0.556400000010485 ]
  mass: 12.010700
- symbol: C  # 30
  coordinates: [  0.319709999997832,  0.230840000009162,  0.355080000007926 ]
  mass: 12.010700
- symbol: C  # 31
  coordinates: [  0.255380000001588,  0.140220000013505,  0.196920000010405 ]
  mass: 12.010700
- symbol: C  # 32
  coordinates: [  0.218459999998039,  0.234160000004458,  0.006480000013476 ]
  mass: 12.010700
- symbol: C  # 33
  coordinates: [  0.160339999998765,  0.142900000013312,  0.820920000007827 ]
  mass: 12.010700
- symbol: C  # 34
  coordinates: [  0.124259999997101,  0.239200000005562,  0.632560000012106 ]
  mass: 12.010700
- symbol: C  # 35
  coordinates: [  0.066419999998455,  0.148560000014774,  0.444920000015725 ]
  mass: 12.010700
- symbol: C  # 36
  coordinates: [  0.029949999998012,  0.244740000004930,  0.257280000011090 ]
  mass: 12.010700
- symbol: C  # 37
  coordinates: [  0.971779999999580,  0.153620000005554,  0.071320000007154 ]
  mass: 12.010700
- symbol: C  # 38
  coordinates: [  0.934850000001535,  0.249360000007686,  0.884080000003710 ]
  mass: 12.010700
- symbol: C  # 39
  coordinates: [  0.876430000001970,  0.158340000006234,  0.698760000003020 ]
  mass: 12.010700
- symbol: C  # 40
  coordinates: [  0.839779999999217,  0.254819999999538,  0.512119999999596 ]
  mass: 12.010700
- symbol: C  # 41
  coordinates: [  0.781910000001076,  0.164920000002577,  0.323200000008003 ]
  mass: 12.010700
- symbol: C  # 42
  coordinates: [  0.746100000000208,  0.264820000004853,  0.140240000013839 ]
  mass: 12.010700
- symbol: H  # 43
  coordinates: [  0.402229999997723,  0.361260000004327,  0.959720000003267 ]
  mass: 1.007940
- symbol: H  # 44
  coordinates: [  0.619609999997913,  0.154760000009292,  0.338520000009840 ]
  mass: 1.007940
- symbol: H  # 45
  coordinates: [  0.439190000000599,  0.742279999999387,  0.682720000010008 ]
  mass: 1.007940
- symbol: H  # 46
  coordinates: [  0.656649999999444,  0.535720000008777,  0.061800000009214 ]
  mass: 1.007940
- symbol: H  # 47
  coordinates: [  0.732440000000435,  0.760620000008176,  0.313000000001214 ]
  mass: 1.007940
- symbol: H  # 48
  coordinates: [  0.697999999997876,  0.870260000003082,  0.536080000008350 ]
  mass: 1.007940
- symbol: H  # 49
  coordinates: [  0.700060000000592,  0.630199999999048,  0.785319999996789 ]
  mass: 1.007940
- symbol: H  # 50
  coordinates: [  0.754950000000814,  0.561060000005899,  0.581200000009257 ]
  mass: 1.007940
- symbol: H  # 51
  coordinates: [  0.785269999997941,  0.856900000006177,  0.917720000011145 ]
  mass: 1.007940
- symbol: H  # 52
  coordinates: [  0.842019999998906,  0.800939999997730,  0.709680000003903 ]
  mass: 1.007940
- symbol: H  # 53
  coordinates: [  0.800129999998881,  0.607380000004147,  0.113800000011988 ]
  mass: 1.007940
- symbol: H  # 54
  coordinates: [  0.863390000002445,  0.569780000004119,  0.920519999999108 ]
  mass: 1.007940
- symbol: H  # 55
  coordinates: [  0.938970000001631,  0.810179999998450,  0.104200000008820 ]
  mass: 1.007940
- symbol: H  # 56
  coordinates: [  0.874979999999669,  0.846279999998870,  0.296040000006466 ]
  mass: 1.007940
- symbol: H  # 57
  coordinates: [  0.893860000002386,  0.601040000000916,  0.484440000004740 ]
  mass: 1.007940
- symbol: H  # 58
  coordinates: [  0.958060000000816,  0.566080000002461,  0.293000000000855 ]
  mass: 1.007940
- symbol: H  # 59
  coordinates: [  0.032819999997781,  0.805280000005002,  0.480680000010979 ]
  mass: 1.007940
- symbol: H  # 60
  coordinates: [  0.968830000001156,  0.840139999999708,  0.672920000002731 ]
  mass: 1.007940
- symbol: H  # 61
  coordinates: [  0.988560000000253,  0.595620000003640,  0.861239999996621 ]
  mass: 1.007940
- symbol: H  # 62
  coordinates: [  0.052229999996922,  0.560540000009680,  0.667640000016819 ]
  mass: 1.007940
- symbol: H  # 63
  coordinates: [  0.127119999997037,  0.800060000002409,  0.852680000004968 ]
  mass: 1.007940
- symbol: H  # 64
  coordinates: [  0.063649999997005,  0.835160000001661,  0.046840000004235 ]
  mass: 1.007940
- symbol: H  # 65
  coordinates: [  0.083919999997695,  0.590980000001866,  0.235360000000385 ]
  mass: 1.007940
- symbol: H  # 66
  coordinates: [  0.147180000001259,  0.555860000004303,  0.040240000003046 ]
  mass: 1.007940
- symbol: H  # 67
  coordinates: [  0.222529999998975,  0.795120000009485,  0.225160000005054 ]
  mass: 1.007940
- symbol: H  # 68
  coordinates: [  0.159139999997598,  0.830560000008842,  0.419640000007607 ]
  mass: 1.007940
- symbol: H  # 69
  coordinates: [  0.178749999998712,  0.584420000002669,  0.605000000007141 ]
  mass: 1.007940
- symbol: H  # 70
  coordinates: [  0.242789999999833,  0.551200000010488,  0.412200000006537 ]
  mass: 1.007940
- symbol: H  # 71
  coordinates: [  0.317099999999028,  0.790300000006482,  0.602880000000020 ]
  mass: 1.007940
- symbol: H  # 72
  coordinates: [  0.253269999999713,  0.822720000008094,  0.796440000002139 ]
  mass: 1.007940
- symbol: H  # 73
  coordinates: [  0.352679999998427,  0.696920000007705,  0.021040000005834 ]
  mass: 1.007940
- symbol: H  # 74
  coordinates: [  0.273319999998765,  0.572240000009401,  0.976960000014008 ]
  mass: 1.007940
- symbol: H  # 75
  coordinates: [  0.340120000001504,  0.546160000007053,  0.787600000016285 ]
  mass: 1.007940
- symbol: H  # 76
  coordinates: [  0.360830000000132,  0.026760000008495,  0.485440000008395 ]
  mass: 1.007940
- symbol: H  # 77
  coordinates: [  0.326409999997236,  0.136400000011936,  0.708600000011651 ]
  mass: 1.007940
- symbol: H  # 78
  coordinates: [  0.303849999997698,  0.335920000006802,  0.440360000004900 ]
  mass: 1.007940
- symbol: H  # 79
  coordinates: [  0.358739999997920,  0.266820000010972,  0.236200000009128 ]
  mass: 1.007940
- symbol: H  # 80
  coordinates: [  0.273540000000403,  0.040100000009451,  0.103840000015002 ]
  mass: 1.007940
- symbol: H  # 81
  coordinates: [  0.216789999999438,  0.096060000006652,  0.311880000002791 ]
  mass: 1.007940
- symbol: H  # 82
  coordinates: [  0.195420000001236,  0.327220000012651,  0.101040000011567 ]
  mass: 1.007940
- symbol: H  # 83
  coordinates: [  0.258669999999631,  0.289600000008554,  0.907720000002169 ]
  mass: 1.007940
- symbol: H  # 84
  coordinates: [  0.183799999999179,  0.050700000005295,  0.725520000011571 ]
  mass: 1.007940
- symbol: H  # 85
  coordinates: [  0.119829999996881,  0.086820000012910,  0.917440000019268 ]
  mass: 1.007940
- symbol: H  # 86
  coordinates: [  0.100709999998200,  0.330920000007342,  0.728640000016405 ]
  mass: 1.007940
- symbol: H  # 87
  coordinates: [  0.164909999996630,  0.295980000007547,  0.537160000008400 ]
  mass: 1.007940
- symbol: H  # 88
  coordinates: [  0.089959999997524,  0.056860000007385,  0.348640000009246 ]
  mass: 1.007940
- symbol: H  # 89
  coordinates: [  0.025960000001067,  0.091720000009069,  0.540880000022391 ]
  mass: 1.007940
- symbol: H  # 90
  coordinates: [  0.006549999996589,  0.336460000003250,  0.353920000013975 ]
  mass: 1.007940
- symbol: H  # 91
  coordinates: [  0.070229999998427,  0.301380000003452,  0.160360000015356 ]
  mass: 1.007940
- symbol: H  # 92
  coordinates: [  0.995160000001339,  0.061840000002721,  0.974680000002460 ]
  mass: 1.007940
- symbol: H  # 93
  coordinates: [  0.931670000001643,  0.096940000004683,  0.168760000007009 ]
  mass: 1.007940
- symbol: H  # 94
  coordinates: [  0.911630000002422,  0.341140000001220,  0.981240000006226 ]
  mass: 1.007940
- symbol: H  # 95
  coordinates: [  0.974880000000817,  0.305999999999588,  0.786120000012370 ]
  mass: 1.007940
- symbol: H  # 96
  coordinates: [  0.899650000001083,  0.066400000000931,  0.601880000012611 ]
  mass: 1.007940
- symbol: H  # 97
  coordinates: [  0.836259999999705,  0.101920000006173,  0.796320000000085 ]
  mass: 1.007940
- symbol: H  # 98
  coordinates: [  0.816029999998343,  0.345820000008068,  0.609200000013551 ]
  mass: 1.007940
- symbol: H  # 99
  coordinates: [  0.880069999999463,  0.312500000000096,  0.416400000012692 ]
  mass: 1.007940
- symbol: H  # 100
  coordinates: [  0.805519999998967,  0.074180000005701,  0.225160000012389 ]
  mass: 1.007940
- symbol: H  # 101
  coordinates: [  0.741689999999652,  0.106780000006495,  0.418640000014929 ]
  mass: 1.007940
- symbol: H  # 102
  coordinates: [  0.706159999999412,  0.200020000002255,  0.000320000005999 ]
  mass: 1.007940
- symbol: H  # 103
  coordinates: [  0.718740000001336,  0.350860000007222,  0.233600000013864 ]
  mass: 1.007940
- symbol: H  # 104
  coordinates: [  0.785549999998569,  0.324620000007479,  0.044320000007084 ]
  mass: 1.007940
- symbol: S  # 105
  coordinates: [  0.566219999999150,  0.315760000001813,  0.709120000002047 ]
  mass: 32.065000
- symbol: S  # 106
  coordinates: [  0.436429999998981,  0.979660000005739,  0.161319999997732 ]
  mass: 32.065000
- symbol: S  # 107
  coordinates: [  0.622359999999699,  0.917379999998672,  0.860000000006005 ]
  mass: 32.065000
- symbol: S  # 108
  coordinates: [  0.492609999998857,  0.581300000007082,  0.312120000006457 ]
  mass: 32.065000

phonon:
